### DISCUSSION TITLE
use-gesture & react-window (or another virtualization lib)

### DISCUSSION THREAD

**Discussion by m7kvqbe1** [pmndrs/use-gesture#619] [Q&A]
Has anyone had success combining use-gesture with a virtualization library like react-window?

I'm attempting to integrate the two and seem to be having issues.

The useDrag handler only seems to get invoked for an initial click on my draggable element.

Any prior art or information about anecdotal usage of the two libraries together would be much appreciated.

**Comment by dbismut** [comment]
Well, any sandbox attempt would be great for a start.  You have no idea how time consuming it is for people trying to help to just set up a sandbox, installing the libs and figuring out what's the usecase you have in mind. Pave the way!

**Reply by m7kvqbe1** [reply]
Thanks for the reply @dbismut. I'll create a minimal repro in a sandbox today.

I was just trying to gauge if it was already a known incompatibility.
